Components/Alerts/alert.css

```diff
@@ -0,0 +1,64 @@
+
+.alert {
+  box-sizing: border-box;
+  width: 100%;
+  border-radius: 5px;
+  padding: 1.25rem 2.5rem;
+  box-shadow: 0 0 15px 5px #ccc;
+  border-left: 0.5rem solid;
```

Comment:
using rem & px for small values.. stick to one .. and for less than 20px you can use px.

Reply:
Will make the changes, thanks